src/network-layer/default/RelayDefaultNetworkLayer.js

```diff
@@ -89,7 +89,8 @@ class RelayDefaultNetworkLayer {
         }
       }).catch(
         error => request.reject(error)
-      )
+      ),
+      request
```

Comment:
Unless I missed something, the newly added `request` is the second argument to the `requests.map(...)` call on line 70. If that's the case, this should not change the behavior as the arrow function is automatically bound to the parents scope.